modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/dtx/AlreadyKnownDtxException.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.andes.kernel.dtx;
+
+import org.wso2.andes.server.txn.DtxException;
+
+import javax.transaction.xa.Xid;
+
+
+public class AlreadyKnownDtxException extends DtxException
```

Comment:
:smile: This is thrown when the we try to start a already started transaction. Can change the name if this is confusing. WDYT?